sdk/batch/Microsoft.Azure.Batch/src/Microsoft.Azure.Batch.csproj

```diff
@@ -14,7 +14,6 @@
   <PropertyGroup>
     <TargetFrameworks>$(RequiredTargetFrameworks)</TargetFrameworks>
     <GeneratePackageOnBuild>True</GeneratePackageOnBuild>
-    <PackageReleaseNotes>For detailed release notes, see: https://aka.ms/batch-net-dataplane-changelog </PackageReleaseNotes>
```

Comment:
If you remove this, will there not be a link to the changelog in our nuget package? Or is there some magic happening to where nuget is rendering the actual CHANGELOG.md file now?